dannce/engine/generator.py

```diff
@@ -1687,6 +2205,16 @@ def visualize(self, original, augmented):
         input("Press Enter to continue...")
 
     def random_continuous_rotation(self, X, y_3d, max_delta=5):
+        """Summary
+
+        Args:
+            X (TYPE): Description
+            y_3d (TYPE): Description
```

Comment:
`y_3d (np.ndarray): 3d target (for MAX network) or voxel center grid (for AVG network)`

Reply:
```suggestion
            y_3d (np.ndarray): 3d target (for MAX network) or voxel center grid (for AVG network)
```